message/client.go

```diff
@@ -11,17 +11,19 @@ type Client struct {
 	endpoint string
 	topic    string
 	region   string
+	ssl      bool
+
 }
 
 // NewClient returns a new Client instance
-func NewClient(endpoint string, topic string, region string) *Client {
-	return &Client{endpoint: endpoint, topic: topic, region: region}
+func NewClient(endpoint string, topic string, region string, ssl bool) *Client {
+	return &Client{endpoint: endpoint, topic: topic, region: region, ssl: ssl}
```

Comment:
@justinlittman this looks like it may be a valuable change. Could you do a separate PR for this? I'm happy to if you are onto other things.